Core/GDCore/Extensions/Builtin/SceneExtension.cpp

```diff
@@ -136,6 +136,20 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsSceneExtension(
       .AddCodeOnlyParameter("currentScene", "")
       .AddParameter("yesorno", _("Deactivate input when focus is lost"))
       .MarkAsAdvanced();
+
+  extension
+      .AddCondition(
+          "HasGameJustResumed",
+          _("Game resumed"),
+          _("Check if the game has just resumed (from being hidden). It "
+            "happens when the game tab is selected, a minimized window is "
+            "restored or the application is put back on front."),
+          _("Game has resumed"),
```

Comment:
```suggestion
          _("Game has just resumed"),
```

convention is "just" === "works only during this frame".